ui/pages/settings/networks-tab/networks-list/networks-list.js

```diff
@@ -9,22 +16,73 @@ const NetworksList = ({
   networkDefaultedToProvider,
   selectedRpcUrl,
 }) => {
+  const t = useI18nContext();
+  const [networks, setNetworks] = useState([]);
```

Comment:
To make clear that these networks are the results of the search, perhaps rename to `searchedNetworks` and `setSearchedNetworks`

Reply:
@danjm Fixed in: 2179697b44f506d161e5258c52789f632ae8bda8